Automattic/wp-calypso
# Add migration instructions basic structure

Related to #91776

## Proposed Changes

* Partially refactor the current Launchpad step, extracting parts to be reused in other Launchpads, like this new step. So we avoid duplicating code.
  * Anyway, I tried to change the minimum that I could in the Launchpad in order to avoid breaking something that we didn't realize. So, for example, the extracted logic receives classNames, so we pass them when calling the new component in the existing launchpad.
  * I also reviewed the spacings in the new classes, but for the current Lanchpad, it's overriding with the existing styles to also avoid anything we could miss and break.
* Create a basic structure for the Migration Instructions step.

<img width="1279" alt="Screenshot 2024-06-19 at 19 21 16" src="https://github.com/Automattic/wp-calypso/assets/876340/160c6568-2c66-4ecf-8457-ae0748e07ec6">

## Known issues

* https://github.com/Automattic/wp-calypso/issues/91964

## Testing Instructions



* Activate the feature flag: `migration-flow/new-migration-instructions-step`.
* Navigate through the flow to import a site, and in the final step make sure you see the new migration instructions screen.
* Check that this screen is responsive and the page works properly.
* Create a new site, and make sure the current Launchpad continues working as previously (you can open the same page on https://wordpress.com/ and on http://calypso.localhost:3000/ to compare). `/setup/build/launchpad?siteSlug={SITE_SLUG}&siteId={SITE_ID}&showLaunchpad=true`

## Pre-merge Checklist
